README.md

```diff
@@ -1,469 +1,3 @@
-
-> [!NOTE]
-> Running `vite dev` or `vite build` without providing `CLOUDFLARE_ENV` will use the default top-level environment.
-> The value of `MY_VAR` will therefore be `'Top-level var'`.
-> As Worker environments are applied at dev and build time, specifying `CLOUDFLARE_ENV` when running `vite preview` or `wrangler deploy` will have no effect.
-
-### Combining Worker environments and Vite modes
-
-You may wish to combine the concepts of [Worker environments](https://developers.cloudflare.com/workers/wrangler/environments/) and [Vite modes](https://vite.dev/guide/env-and-mode.html#modes).
-With this approach, the mode can be used to select the environment and a single method can be used to determine environment specific configuration and code.
-Consider again the previous example:
-
-```toml
-# wrangler.toml
-
-name = "my-worker"
-compatibility_date = "2024-12-30"
-main = "./src/index.ts"
-
-vars = { MY_VAR = "Top-level var" }
-
-[env.staging]
-vars = { MY_VAR = "Staging var" }
-
-[env.production]
-vars = { MY_VAR = "Production var" }
-```
-
-Next, provide `.env.staging` and `.env.production` files:
-
-```sh
-# .env.staging
-
-CLOUDFLARE_ENV=staging
-```
-
-```sh
-# .env.production
-
-CLOUDFLARE_ENV=production
-```
-
-By default, `vite build` uses the 'production' mode.
-Vite will therefore load the `.env.production` file to get the environment variables that are used in the build.
-Since the `.env.production` file contains `CLOUDFLARE_ENV=production`, the Cloudflare Vite plugin will select the 'production' Worker environment.
-The value of `MY_VAR` will therefore be `'Production var'`.
-If you run `vite build --mode staging` then the 'staging' mode will be used and the 'staging' environment will be selected.
-The value of `MY_VAR` will therefore be `'Staging var'`.
-
-## Migrating from `wrangler dev`
-
-Migrating from `wrangler dev` is a simple process and you can follow the instructions in the [Quick start](#quick-start) to get started.
-There are a few key differences to highlight:
-
-### Input and output Worker config files
-
-In the Vite integration, your Worker config file (for example, `wrangler.toml`) is the input configuration and a separate output configuration is created as part of the build.
-This output file is a snapshot of your configuration at the time of the build and is modified to reference your build artifacts.
-It is the configuration that is used for preview and deployment.
-
-### Redundant fields in the Wrangler config file
-
-There are various options in the Worker config file that are ignored when using Vite, as they are either no longer applicable or are replaced by Vite equivalents.
-If these options are provided, then warnings will be printed to the console with suggestions for how to proceed.
-Examples where the Vite configuration should be used instead include `alias` and `define`.
+This repository has been migrated to https://github.com/cloudflare/workers-sdk/tree/main/packages/vite-plugin-cloudflare.
```

Comment:
Or should it be

```suggestion
This repository has migrated to https://github.com/cloudflare/workers-sdk/tree/main/packages/vite-plugin-cloudflare.
```

Reply:
I think the passive "has been" is fine in this case.